charts/etcd/templates/etcd-role.yaml

```diff
@@ -36,4 +36,14 @@ rules:
   - list
   - patch
   - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - patch
+  - update
```

Comment:
Why is `patch` and `update` required?

Reply:
You're right, `patch` and `update` aren't required
Thanks for pointing out :)